flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/TransformationsTest.java

```diff
@@ -83,46 +80,6 @@ public void testLegacyStreamSource() {
         assertTrue(sourceTransform.getOperator().emitsProgressiveWatermarks());
     }
 
-    @Test
-    public void testStreamTransformationScanProvider() {
-        final JavaStreamTableTestUtil util = javaStreamTestUtil();
-        final StreamTableEnvironment env = util.tableEnv();
-
-        final Table table =
-                env.from(
-                        TableDescriptor.forConnector("values")
-                                .option("bounded", "false")
-                                .schema(dummySchema())
-                                .build());
-
-        final Transformation<RowData> transformation =
-                env.toChangelogStream(table)
-                        .<RowData>map(r -> new GenericRowData(0))
-                        .getTransformation();
-
-        assertFalse(TransformationScanProvider.of(transformation, "scan").isBounded());
-    }
-
-    @Test
-    public void testBatchTransformationScanProvider() {
```

Comment:
the tests are important, they test the propagation of boundedness. please readd them

Reply:
as discussed offline, we can remove them as they're only testing mocks now